pyproject.toml

```diff
@@ -49,6 +60,7 @@ profile = "black"
 
 [tool.pytest.ini_options]
 addopts = "--black --cov=invenio_edugain --cov-report=term-missing"
+live_server_scope = "module"
```

Comment:
why is that necessary?

Reply:
for context: it's an configuration option used by `pytest-flask` (see [its docs](https://discord.com/channels/@me/861929127728644096/1352211480665260044))
it *scopes the server fixture to modules* (default is session scope)

turns out this isn't necessary here

Removed.